src/Admin/AbstractAdmin.php

```diff
@@ -3953,7 +3954,13 @@ final protected function appendParentObject(object $object): void
                 $propertyAccessor = PropertyAccess::createPropertyAccessor();
                 $propertyPath = new PropertyPath($this->getParentAssociationMapping());
 
-                $value = $propertyAccessor->getValue($object, $propertyPath);
+                try {
+                    $value = $propertyAccessor->getValue($object, $propertyPath);
+                } catch (AccessException $e) {
```

Comment:
what happens if the entity does not have the getParent method? Would this try catch hide some potential problems?

Reply:
PropertyAccessor throws this exception in 3 cases:

1. The getter is used but a native PHP `\Error` was catched by PropertyAccessor, the PHP version is >= 7.4 and the message of the error is "Typed property .... must not be accessed before initialization"
1. The property is public and PropertyAccessor checks if is was set (initialised)
1. The getter is used but a native PHP `\TypeError` was catched by PropertyAccessor and the message of the error is "Return value ... must be of type ..., null returned" (in which case the property is untyped and uninitialised and the getter return type is not nullable)

> what happens if the entity does not have the getParent method? Would this try catch hide some potential problems?

In case the property exists and is private/protected but the `getParent` method does not exist (or is private/protected) a `\Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException` is thrown.

Reply:
> NoSuchPropertyException

`NoSuchPropertyException extends AccessException` so it's caught too by your try/catch.
Same for `NoSuchIndexException`.

Maybe we should add a check `if ($e instanceof NoSuchPropertyException || $e instanceof NoSuchIndexException) throw $e`.

Reply:
> I would say yes because "noSuchPropertyException" extends AccessException...
> 
> Dunno if we can filter the right case we want until support for Sf4.4 is dropped.
> 
> Should we check the message of the exception until the use of the new exception ?

Ok... sharp, I've overlooked that "detail" 😄 

I could add:

```php
if (!$e instanceof UninitializedPropertyException && get_class($e) !== AccessException::class) {
    throw $e; // Re-throw. We only want to "ignore" AccessException (Sf < 5.1) and UninitializedPropertyException (Sf >= 5.1)
}
```

Reply:
Yes, I think it would be better with this re-throw

Reply:
Linter are failing

Reply:
I fixed this one thing in my code, but now PHP CS Fixer is failing on other things. It is using PHP CS Fixer 3.5.0 which has been released 9 hours ago, so maybe this is a bug in that new release